### PR TITLE
fix: `No options` dropdown background appears white in dark mode 

### LINKED ISSUE #312: Bug: `No options` dropdown background appears white in dark mode
### URL(s)

https://eslint.org/play/

### What did you do?

1. Switch to `dark` mode in playground
2. Select all the options from any multi select dropdown until nothing left.

### What did you expect to happen?

image
image

### What actually happened?

image

Screenshot 2022-08-13 124514

### Participation

- [X] I am willing to submit a pull request for this issue.

### Additional comments

_No response_

### PR DESCRIPTION
**Pull request by shanpriyan**
Resolves #312

### CODE REVIEW

File: src/playground/components/Configuration.js [hunk 1]
```diff
@@ -64,6 +64,15 @@ const customStyles = {
...
         backgroundColor: "var(--lighter-background-color)"
+    }),
+    noOptionsMessage: styles => ({
+        ...styles,
+        backgroundColor: "var(--body-background-color)"
+    }),
+    menuList: styles => ({
+        ...styles,
+        backgroundColor: "var(--body-background-color)",
+        border: "1px solid var(--border-color)"
```
Comment (harish-sethuraman) [inline src/playground/components/Configuration.js:75]: Already there is a border for all options? Why are we introducing border again for the menuList?
| Before (main branch) | After  (your branch) | 
| ---- | ---- | 
| Screenshot 2022-08-13 at 1 25 17 PM | Screenshot 2022-08-13 at 1 25 29 PM |

Comment (snitin315) [reply src/playground/components/Configuration.js:75]: Yes, we can remove the border here.

```suggestion
```

Comment (shanpriyan) [reply src/playground/components/Configuration.js:75]: image
Without the border property in `menuList`, `No options` will appear with no border.

Will fix it.

**Comment by amareshsm** [comment]
@shanpriyan Please add before/After screenshot.

**Comment by shanpriyan** [comment]
> @shanpriyan Please add a before/After screenshot.

| Before fix (main) | After fix (my branch) |
|:--------|:-------|
|image|image

**Review by amareshsm — APPROVED** [review]

**Comment by amareshsm** [comment]
Looks good to me. Thanks

**Review by kecrily — APPROVED** [review]
LGTM. Thanks for your contribution ✨

**Comment by kecrily** [comment]
I noticed that after Strek reopened the issue, CI was working fine. But I don't know why this is happening, can someone please explain?

**Comment by harish-sethuraman** [comment]
We had build failure in the main branch so the build was failing inside the lint job. I had sent a PR to fix the build issue and was merged to main so now it is working fine after re triggering.

**Review by harish-sethuraman — APPROVED** [review]
LGTM

**Comment by kecrily** [comment]
Oh, I see. I also tried re-run before and it still failed. Now I think that re-run should be based on the original environment, not the new one.

**Pull request merged**